ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -53,6 +53,15 @@ declare admin_iface_active
 declare install_to_hdd
 declare nicsup_done
 
+# Additional NIC tags definitions
+declare -A nictag_macs
+declare -A nictag_ifaces
+declare -A nictag_IPs
+declare -A nictag_netmasks
+declare -A nictag_vlans
+
+ADVANCED_INSTALL=1
```

Comment:
1. Changing case
2. If we want to properly support response files with unattended advanced installation, it needs to default to true (1). See [here](https://github.com/erigones/esdc-factory/blob/167a68bfcf79610240b188165aa26930e6c9b4e3/ansible/templates/usb/scripts/prompt-config.sh.j2#L1363).